client/extensions/woocommerce/state/action-types.js

```diff
@@ -14,6 +14,9 @@ export const WOOCOMMERCE_PRODUCT_CREATE_SUCCESS = 'WOOCOMMERCE_PRODUCT_CREATE_SU
 export const WOOCOMMERCE_PRODUCT_EDIT = 'WOOCOMMERCE_PRODUCT_EDIT';
 export const WOOCOMMERCE_PRODUCT_UPDATED = 'WOOCOMMERCE_PRODUCT_UPDATED';
 export const WOOCOMMERCE_PRODUCT_VARIATION_EDIT = 'WOOCOMMERCE_PRODUCT_VARIATION_EDIT';
+export const WOOCOMMERCE_REQUEST = 'WOOCOMMERCE_REQUEST';
```

Comment:
I didn't understand that the ultimate/long-term intention was to have a single action cover multiple APIs so yeah, not including JETPACK in the action identifier makes sense to me.  Adding RESTAPI to the identifier instead is a good idea.